### ISSUE TITLE
Prettier error not showing on typescript project

### ISSUE THREAD

**Issue by mandaputtra** [ChristianChiarulli/LunarVim#307]
## Step to reproduce

1. Follow the step on README.md to install `typescript` with `LspInstall:`
2. Changing `efm-general.lua` to this

```
require"lspconfig".efm.setup {
    -- init_options = {initializationOptions},
    cmd = {DATA_PATH .. "/lspinstall/efm/efm-langserver"},
    init_options = {documentFormatting = true, codeAction = false},
    filetypes = {"lua", "python", "typescript", "javascriptreact", "javascript", "sh", "html", "css", "json", "yaml", "markdown", "vue"},
    settings = {
        rootMarkers = {".git/"},
        languages = {
            python = python_arguments,
            lua = lua_arguments,
            sh = sh_arguments,
            javascript = tsserver_args,
            javascriptreact = tsserver_args,
            html = {prettier},
            css = {prettier},
            json = {prettier},
            yaml = {prettier},
            markdown = {markdownPandocFormat},
	    typescript = {prettier, eslint}
            -- javascriptreact = {prettier, eslint},
            -- javascript = {prettier, eslint},
            -- markdown = {markdownPandocFormat, markdownlint},
        }
    }
}

```

3. Open typescript repository that had eslint and prettier in it
4. Error on typescript side are shown but not on prettier side (indentation error format etc)

Actually a _dirty_ solution already provided by @rebuilt [here](https://github.com/ChristianChiarulli/LunarVim/issues/299#issuecomment-827296578) :+1: :bowing_man: . I create this issue so @rebuilt can reference it during pull request. 

## Some info

```
NVIM v0.5.0-dev+1273-g8402865cb
Ubuntu 20.10
```

**Comment by mandaputtra** [comment]
I'm checking in again, I test it with Vue. It also requires the _dirty_ solution to make prettier working properly.

```lua
--  efm-general.lua
require"lspconfig".efm.setup {
    -- init_options = {initializationOptions},
    cmd = {DATA_PATH .. "/lspinstall/efm/efm-langserver"},
    init_options = {documentFormatting = true, codeAction = false},
    filetypes = {"lua", "python", "typescript", "javascriptreact", "javascript", "sh", "html", "css", "json", "yaml", "markdown", "vue"},
    settings = {
        rootMarkers = {".git/"},
        languages = {
            python = python_arguments,
            lua = lua_arguments,
            sh = sh_arguments,
            javascript = tsserver_args,
            javascriptreact = tsserver_args,
            html = {prettier},
            css = {prettier},
            json = {prettier},
            yaml = {prettier},
            markdown = {markdownPandocFormat},
	    typescript = {prettier, eslint},
            vue = {prettier, eslint},
            -- javascriptreact = {prettier, eslint},
            -- javascript = {prettier, eslint},
            -- markdown = {markdownPandocFormat, markdownlint},
        }
    }
}
```

And then add this.

```lua
-- lv-autocommands/init.lua
local typescript_format = {'BufWritePre', '*.ts', 'lua vim.lsp.buf.formatting_sync(nil, 1000)'}
table.insert(auto_formatters,typescript_format)
```

I also want to ask some silly question, what with the Chinese icon there? Is it error marking? 

![Screenshot from 2021-04-27 20-48-35](https://user-images.githubusercontent.com/23342943/116253154-6c608880-a79a-11eb-8f72-0cdea801e1fd.png)

**Comment by rebuilt** [comment]
You have to install a nerd font so you can see all the symbols properly.  

If you're running arch it's easier than other distros and you can follow the instructions below.  Other wise go to the nerd font website and follow theirs.

Choose a font from this list
```bash
nerd-fonts-cascadia-code, nerd-fonts-fira-code, nerd-fonts-hack, nerd-fonts-iosevka, nerd-fonts-jetbrains-mono, nerd-fonts-overpass, nerd-fonts-roboto-mono, nerd-fonts-source-code-pro, nerd-fonts-terminus, nerd-fonts-ubuntu-mono
```
And then install it with (I'm assuming arch since you installed an arch package)
```bash
sudo pacman -S nerd-fonts-fira-code
```
To make sure you've correctly installed your font
```bash
fc-list | grep -i "Fira"
```
Then you have to configure your terminal. Make sure your terminal supports nerd fonts, i.e. terminator, alacritty, kitty.

This is the font section of my alacritty config: ~/.config/alacritty/alacritty.yml
```bash
# Font configuration (changes require restart)
font:
  # The size to use.
  size: 14
  #  The normal (roman) font face to use.
  normal:
    family: Fira Code
    # Style can be specified to pick a specific face.
    style: Medium

  # The bold font face
  bold:
    family: Fira Code
    # Style can be specified to pick a specific face.
    style: Bold

  # # The italic font face
  italic:
    family: Fira Code
    # Style can be specified to pick a specific face.
    style: Italic
```

**Comment by mandaputtra** [comment]
Closed on #311
